### PR TITLE
console,jsre:use goja as the js Interpreter instead of otto

### PR DESCRIPTION
**Pull request by ShadowErii**
fix the sort problem,but not finish with the welcome.
截屏2020-07-21 下午6 50 21

### CODE REVIEW

File: console/console.go [hunk 1]
```diff
@@ -108,120 +110,141 @@ func New(config Config) (*Console, error) {
...
+
+// initExtensions loads and registers web3.js extensions.
+func (c *Console) initExtensions() error {
+	// Compute aliases from server-provided modules.
 	apis, err := c.client.SupportedModules()
 	if err != nil {
 		return fmt.Errorf("api modules: %v", err)
 	}
-	flatten := "var ctxc = web3.ctxc; var personal = web3.personal; "
+	aliases := map[string]struct{}{"cxtc": {}, "personal": {}}
```
Comment (ucwong) [inline console/console.go:193]: typo

Comment (ShadowErii) [reply console/console.go:193]: got it

File: console/console_test.go [hunk 2]
```diff
@@ -0,0 +1,327 @@
...
+
+	// Create a networkless protocol stack and start an Cortex service within
+	stack, err := node.New(&node.Config{DataDir: workspace, UseLightweightKDF: true, Name: testInstance})
+	if err != nil {
+		t.Fatalf("failed to create node: %v", err)
+	}
+	ctxcConf := &ctxc.Config{
+		//TODO:change to devloper
+		Genesis: core.DefaultGenesisBlock(),
+		Miner: miner.Config{
```
Comment (ShadowErii) [inline console/console_test.go:86]: add miner.coinbase here,but can't get it by ctxc.coinbase.

Comment (ucwong) [reply console/console_test.go:86]:
```
func (miner *Miner) SetCoinbase(addr common.Address) {
        miner.coinbase = addr
        miner.worker.setCoinbase(addr)
}
```

Comment (ucwong) [reply console/console_test.go:86]: coinbase can be only set by SetCoinbase

Comment (ucwong) [reply console/console_test.go:86]: Etherbase = Coinbase

Comment (ShadowErii) [reply console/console_test.go:86]: But I've already set it up through ctxcConf.miner.coinbase

Comment (ucwong) [reply console/console_test.go:86]: > But I've already set it up through ctxcConf.miner.coinbase，doesn't that work?

Also right

File: console/console.go [hunk 3]
```diff
@@ -30,15 +30,16 @@ import (
...
 	"github.com/CortexFoundation/CortexTheseus/rpc"
+	"github.com/dop251/goja"
 	"github.com/mattn/go-colorable"
 	"github.com/peterh/liner"
-	"github.com/robertkrimen/otto"
 )
 
 var (
-	passwordRegexp = regexp.MustCompile(`personal.[nusi]`)
+	passwordRegexp = regexp.MustCompile(`personal.[nus]`)
```
Comment (ucwong) [inline console/console.go:42]: nusi is better including personal.import

**Pull request merged**